lib/redux.js

```diff
@@ -0,0 +1,14 @@
+const addReduxOption = (containerOptions, componentName) => {
+	const reduxContainerFlags = {
```

Comment:
I guess it wasn't changed in this PR but it might be a great occasion to rename this variable `reduxContainerTags` to be consistent with the `lib/flow.js` file. 😊

Reply:
c1ea1f2